src/formatDate.js

```diff
@@ -50,7 +50,56 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const newFormatDate = [];
+  const initialFormatDate = date.split(fromFormat[3]);
+  const indexOfDay = fromFormat.indexOf('DD');
+  const indexOfMonth = fromFormat.indexOf('MM');
+  let indexOfYear;
+
+  if (fromFormat.indexOf('YYYY') === -1) {
+    indexOfYear = fromFormat.indexOf('YY');
+  } else {
+    indexOfYear = fromFormat.indexOf('YYYY');
+  }
+
+  const actualDate = {
+    DD: initialFormatDate[indexOfDay],
+    MM: initialFormatDate[indexOfMonth],
+    YY: initialFormatDate[indexOfYear],
```

Comment:
Not the best choice of prop names - try to follow naming convention for variables.